Build.ps1

```diff
@@ -2,12 +2,56 @@
 # GraphicalTools includes two modules: Microsoft.PowerShell.GraphicalTools and Microsoft.PowerShell.ConsoleGuiTools
 # To build them all leave -ModuleName off the `InvokeBuild` command (e.g. Invoke-Build Build).
 # To build only one, specify it using the -ModuleName paramater (e.g. Invoke-Build Build -ModuleName Microsoft.PowerShell.ConsoleGuiTools).
+$ModuleName = "Microsoft.PowerShell.ConsoleGuiTools"
+$BuildPath = "$PSScriptRoot/module/$ModuleName"
+$PsdPath = "src/$ModuleName/$ModulePath/$($ModuleName).psd1"
+
+# Assume this is the first build
+$build = 0
+
+$psd1Content = Get-Content $PsdPath -Raw -ErrorAction SilentlyContinue
+if ($psd1Content) {
+    # Extract the ModuleVersion from the .psd1 content using regular expression
+    if ($psd1Content -match "ModuleVersion\s+=\s+'(.*?)'") {
+        $prevVersion = $Matches[1]
+        $prevVersionParts = $prevVersion -split '\.'
+        $build = [int]$prevVersionParts[3] + 1
```

Comment:
Do we need all this? The pipeline isn't going to save any increment to the build version.

Reply:
I need it for local build/test. I'd appreciate keeping it.